### PR TITLE
xia 1.4.0

### PR DESCRIPTION
**Pull request by simonjwright**
xia is XPath in Ada, used for processing XML files.

### CODE REVIEW

File: upload/devel-1.1/index/xi/xia/xia-1.4.0.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name = "xia"
+description = "Implements XPath 1.0 in Ada"
+version = "1.4.0"
+licenses= "GPL-3.0-or-later"
```
Comment (onox) [inline upload/devel-1.1/index/xi/xia/xia-1.4.0.toml]: It seems the project is using the GCC Runtime Library Exception. Shouldn't it therefore contain ` WITH GCC-exception-3.1`?

Comment (simonjwright) [reply upload/devel-1.1/index/xi/xia/xia-1.4.0.toml]: @onox, indeed, but what should the syntax of `licenses` be? (the guidance may have been updated recently, no help when I last looked)

Comment (Fabien-Chouteau) [reply upload/devel-1.1/index/xi/xia/xia-1.4.0.toml]: example here https://github.com/alire-project/alire-index/blob/stable-1.1/index/gn/gnatcoll/gnatcoll-21.0.0.toml

Comment (simonjwright) [reply upload/devel-1.1/index/xi/xia/xia-1.4.0.toml]: Thanks. I guess there’ll be a 1.4.1 release. If it’s OK, I’ll leave the PR running until it shows up all my other mistakes :-)

Comment (mosteo) [reply upload/devel-1.1/index/xi/xia/xia-1.4.0.toml]: Actually, the checks seem to be stuck for no reason, so you can push some change and that might unstuck them.

Comment (simonjwright) [reply upload/devel-1.1/index/xi/xia/xia-1.4.0.toml]: Still stuck?

Comment (mosteo) [reply upload/devel-1.1/index/xi/xia/xia-1.4.0.toml]: Yup. Very strange. I'm going to try from my fork.

Comment (mosteo) [reply upload/devel-1.1/index/xi/xia/xia-1.4.0.toml]: Ah, I see. Your file is at `upload/devel-1.1/index/xi/xia/xia-1.4.0.toml` when it should be at `index/xi/xia/xia-1.4.0.toml`. Just a matter of moving it there and checks should run.

Comment (simonjwright) [reply upload/devel-1.1/index/xi/xia/xia-1.4.0.toml]: That’s where Alr 1.1.0-dev+local-toolchain told me to put it:
```
Publishing assistant: step 6 of 6: Generate index manifest
✓ Your index manifest file has been generated at /Users/simon/Developer/xia-for-alire/alire/releases/xia-1.4.0.toml
ⓘ Please upload this file to https://github.com/simonjwright/alire-index/upload/devel-1.1/index/xi/xia to create a pull request against the community index.
```

Comment (mosteo) [reply upload/devel-1.1/index/xi/xia/xia-1.4.0.toml]: Yup, but in that that URL the `/upload` is a the upload service of github and `devel-1.1` is the branch to which you are going to commit. I see how that can lead to confusion, if you end up doing it manually. Unless you actually visited there? Then I'm not sure what happened.

**Review by mosteo — APPROVED** [review]

**Comment by mosteo** [comment]
Congrats on passing all checks on the first actual attempt, Simon :) Merging.

File: upload/devel-1.1/index/xi/xia/xia-1.4.0.toml (hunk 1 above)
Comment (simonjwright) [reply upload/devel-1.1/index/xi/xia/xia-1.4.0.toml]: "upload service of github"??? news to me, no idea how to use it. Sorry if you included it in the docs ...

**Comment by simonjwright** [comment]
Phew!

The .toml in the index now has a subtly different `licenses` entry from the one in the actual project (i.e. it has the RTLE). Best left, or should I update to version 1.4.1?

**Comment by mosteo** [comment]
As long as the intention is clear I wouldn't fret about it. If a clear reason arises you can always release a new version.

**Pull request merged**